examples/typescript/package.json

```diff
@@ -3,12 +3,13 @@
   "version": "0.0.0",
   "name": "example-typescript",
   "dependencies": {
-    "react": "16.4.1",
-    "react-dom": "16.4.1",
+    "react": "^16.4.0",
+    "react-dom": "^16.4.0",
     "typescript": "*"
   },
   "devDependencies": {
     "@types/jest": "^23.1.1",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
Required by ts-jest.

Reply:
we can probably drop ts-jest from the example now with babel 7. Yes, it has some limitations, but as a quick getting started having something that works with babel is way easier. We can point to ts-jest for people who do not wish to use babel, but we do not need to maintain that example. IMO 🙂